Doc/whatsnew/3.8.rst

```diff
@@ -215,7 +215,7 @@ subdirectories).
 Debug build uses the same ABI as release build
 -----------------------------------------------
 
-Python now uses the same ABI whether it built in release or debug mode. On
+Python now uses the same ABI whether it's built in release or debug mode. On
```

Comment:
I think this should be:

```suggestion
Python now uses the same ABI whether it's built-in release or debug mode. On
```

Reply:
I think the original is correct. 

Compare: "Python is built in release mode."